helfomat-infrastructure-google-maps/src/main/kotlin/de/helfenkannjeder/helfomat/infrastructure/google/maps/GoogleMapsConfiguration.kt

```diff
@@ -0,0 +1,13 @@
+package de.helfenkannjeder.helfomat.infrastructure.google.maps
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.stereotype.Component
+
+/**
+ * @author Valentin Zickner
+ */
+@Component
+@ConfigurationProperties("googlemaps")
+class GoogleMapsConfiguration {
+    var apiKey: String? = null
```

Comment:
if spring can handle it, make this a val and probably not optional

Reply:
it can't :(